frontend/packages/kubevirt-plugin/src/components/vm-status/vm-status.tsx

```diff
@@ -164,7 +181,11 @@ export const VMStatus: React.FC<VMStatusProps> = ({
   const vmiLike = vm || vmi;
 
   const { status, pod, progress, importerPodsStatuses } = vmStatusBundle;
-  const title = t(status.getLabelKey()) || status.toString() || t('kubevirt-plugin~Unknown'); // TODO status.toVerboseString() should be called to pass to popup header
+
+  const title =
+    t(status.getLabelKey()) ||
+    status.toString(getStatusSuffix(vmStatusBundle, t)) ||
```

Comment:
I noticed this is not properly localized so I did not touch this issue in this bug. Are there any plans on fixing this? @yaacov @pcbailey